seed/challenges/01-front-end-development-certification/html5-and-css.json

```diff
@@ -4584,7 +4584,7 @@
         "assert($(\"h1\").hasClass(\"pink-text\"), 'message: Your <code>h1</code> element should have the class <code>pink-text</code>.');",
         "assert($(\"h1\").hasClass(\"blue-text\"), 'message: Your <code>h1</code> element should have the class <code>blue-text</code>.');",
         "assert($(\"h1\").attr(\"id\") === \"orange-text\", 'message: Your <code>h1</code> element should have the id of <code>orange-text</code>.');",
-        "assert(code.match(/<h1[\\s\\S]*?style/gi) && code.match(/<h1[\\s\\S]*?style[\\s\\S]*?color\\s*?:/gi), 'message: Give your <code>h1</code> element the inline style of <code>color&#58; white</code>.');",
+        "assert(code.match(/<h1[\\s\\S]*?style/gi) && code.match(/<h1[\\s\\S]*?style\\s*?=\\s*[\\\"]{1}\\s*\\s*\\S*;*\\s*color\\s*?:\\s*?white\\s*;*\\s*[\\\"]{1}/gi), 'message: Give your <code>h1</code> element the inline style of <code>color&#58; white</code>.');",
```

Comment:
There are still some issues with this PR.
The majority of the issues have to do with the matching of spaces. `\s*` will match zero or more whitespace characters. Anywhere where whitespace is allowed but not required, please use that. 
Currently, a space between `h1` and `style` is optional, meaning that `<h1style=` will pass. `+` matches 1 or more whitespaces, so `\s+` should do the trick there.
I feel confident using RegEx, but I'm not a mastermind just yet. Can you please share what you're trying to do with `[\\\"]{1}`? From what I understand, you should only need to match a quote there.
With this change, I think we no longer need both of the `code.match`'s in the test. You can remove the first one.
Thank you for contributing!